### ISSUE TITLE
Bible Source

### ISSUE THREAD

**Issue by jsarenik** [MMU-TDMLab/CompLex#6]
Hello!

Reading about one used source, The World English Bible, at https://worldenglish.bible/ it seems to me that according to SemEval's [Anti-Harassment policy](http://naacl.org/policies/anti-harassment.html) it would be proper to use the British Edition which uses words LORD or GOD in uppercase instead of the word which may [create discomfort](https://naacl.org/policies/anti-harassment.html#discomfort).

In other words here is a quick fix using `sed`:

```
sed -i 's/Ya[a-z]\+/LORD/g' lcp_single_train.tsv
sed -i 's/Ya[a-z]\+/LORD/g' lcp_multi_train.tsv
```

Thank you for consideration!

**Comment by mattshardlow** [comment]
Hi, thanks for the typo fixes in the pull requests. 

I have some concerns over altering the corpus as follows. I've avoided using the term in question in my response below to avoid further offence or discomfort.

1) There's no way to guarantee that the regex would accurately match in all cases. Certainly the regex you've given would hit a lot of false positives. If we used the full word then cases where there is conjugation, punctuation, possessive apostrophes etc. may be missed. Also, according to the first link, in the translations that don't use God's name this is either translated as 'LORD' or as 'GOD' so translating all instances to LORD would miss those that have been translated as 'GOD'.

2) We're in the middle of the shared task at the moment and editing the corpus would mean that participants are working with different versions. I feel quite strongly that any changes that are made shouldn't be made until after the end of the ST.

3) I wasn't aware that using God's name would create discomfort. Having read up on this, I understand it is a matter of respect for God maintained primarily in Jewish and more recently in Catholic traditions, but not protestantism (the tradition, I come from). It does occur to me that other traditions/denominations may find it offensive to remove what is seen as God's proper name from the texts (e.g., Jehovah's Witnesses). Although I am unsure of this.

4) Further, as the texts are not being used in a religious purpose, I would note that the religious nature of the texts are less. The researcher does not need to actively read, write or engage with the word in question as they would if using these texts for biblical study for example.

5) The word is used as a target token in the training corpus. Replacing it with another word would mean that the annotations attached to that token are no longer correct. So we would have to drop these from the training corpus - again creating  further disruption to the Shared Task.

What I'd suggest as a way forwards is as follows:

1) If you want a version of the corpus that doesn't contain God's name, you are welcome to run the REGEX as above on the trial, training and test set. This shouldn't have too much of an effect on your system's abillity to participate in the ST.
2) After the ST has finished, I would be happy to host a second version of the data with the replacement and the items dropped from the training data, if you are willing to produce and submit a pull request. I would suggest that the correct way to do this is to manually go through the verses containing God's name and replace it with the name given by the translators of the British  Edition of the WEB.
3) I'll put up a disclaimer on the Github and host the modified version in a separate folder with a readme explaining how it differs from the other version.

Please feel free to submit the PR any time and I will approve it once the ST is finished.

**Comment by jsarenik** [comment]
@mattshardlow Wow! Thank you for a thorough answer!

Yes, as for me and my household your suggestions make sense. Deal.
